MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 include README.md
 include LICENSE
-include requirements.txt
+include requirements/requirements.txt
 recursive-exclude * *.py[co]
 include versioneer.py
-include version/_version.py
+include src/_version.py
```

Comment:
Do we need to include this file or shouldn't it get included since we package the "src" folder? I mean, we are not putting in the manifest all *.py in src, right?

Reply:
You could be right about this. This line too had been added by versioneer's configuration but I think it is not necessary.